### PR TITLE
fix(context): request host for update

### PR DESCRIPTION
**Pull request by zeroedin**
## What I did

1. Added request to host for an update after  properties changed in `contextConsumer`

### CODE REVIEW

File: lib/context/color.ts [hunk 1]
```diff
@@ -281,6 +281,7 @@ export class ColorContextConsumer extends ColorContextController implements Reac
         } else {
           this.host.setAttribute(this.attribute, next);
         }
+        this.host.requestUpdate();
```
Comment (bennypowers) [inline lib/context/color.ts:284]: Can you move this to lines 80 and 73?

**Review by bennypowers — CHANGES_REQUESTED** [review]

File: lib/context/color.ts (hunk 1 above)
Comment (zeroedin) [reply lib/context/color.ts:284]: Ok, gotcha as we don't need to request the update on an attribute change ✅

**Review by bennypowers — APPROVED** [review]
Good catch thanks for fixing this

File: .changeset/five-jars-smell.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@rhds/elements": patch
+---
+
+Added request to host for an update after property changed in contextConsumer
```
Comment (bennypowers) [inline .changeset/five-jars-smell.md:5]:
```suggestion
Fixed context features, making sure they update the host element
```

**Pull request merged**